osu.Game.Rulesets.Osu/Objects/Drawables/Connections/FollowPointRenderer.cs

```diff
@@ -100,15 +100,25 @@ private void update()
                             Scale = new Vector2(1.5f * currHitObject.Scale),
                         });
 
-                        using (fp.BeginAbsoluteSequence(fadeInTime))
+                        HasFailed.BindValueChanged(v =>
```

Comment:
this is a memory leak

Reply:
I don't really know much about this, is creating a bindable and binding inside the follow point the correct way to do it?

Reply:
that is one way, yep. but leave this for now as this whole PR may need some more thought (was discussing with @smoogipoo yesterday and we're still undecided on direction).